conda-forge.yml

```
@@ -20,4 +23,5 @@ github_actions:
 provider:
   linux_64: github_actions
```

```suggestion
  linux_64: azure
```

I would like to merge this in, rerender, and merge immediately (to save the dedicated CI resources). We don't need to be so pedantic the CI building especially when many of us are looking at this. 

I can upload the builds to c-f when completed in 12-18 hours.

---

> We don't need to be so pedantic the CI building especially when many of us are looking at this.

I agree, which is why I would just merge this as-is (thanks a lot for building out the linux side!).

The server outage is not a permanent thing, we were just unlucky with the Xmas break (GPU capacity was already reduced to half since December) and now the breakage on a Friday that took the rest offline. But this should be resolved sooner rather than later, and so a switch to azure (where the builds time out anyway) makes little sense to me.

---

ok. as you wish.